src/app.ts

```diff
@@ -27,6 +29,9 @@ export class App {
     });
 
     app.setGlobalPrefix(globalPrefix);
+
+    app.useGlobalPipes(new InputCharacterValidationPipe());
+
```

Comment:
You can pass multiple pipes at once into `useGlobalPipes`, so we can combine this with the line below to be
```
app.useGlobalPipes(
  new InputCharacterValidationPipe(),
  new ValidationPipe({
    whitelist: true,
    transform: true,
  }),
);
```